satpy/etc/readers/iasi_l2_cdr_nc.yaml

```diff
@@ -0,0 +1,20 @@
+reader:
+  name: iasi_l2_cdr_nc
+  short_name: IASI l2 CDR
+  long_name:  IASI All Sky Temperature and Humidity Profiles - Climate Data Record Release 1.1 - Metop-A and -B
+  description: >
+    Reader for IASI All Sky Temperature and Humidity Profiles - Climate Data
+    Record Release 1.1 - Metop-A and -B. Data and documentation are available
+    from http://doi.org/10.15770/EUM_SEC_CLM_0063. Data are also available
+    from the EUMETSAT Data Store under ID EO:EUM:DAT:0576.
+  status: Alpha
+  supports_fsspec: false
```

Comment:
Does it support it now that you subclass from the fsspec version of the netcdf file handler class?

Reply:
Maybe.  I didn't test it.  Not sure if "maybe" is valid here.

Reply:
Changed to True in e518bb2.